x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/data/rule_migrations_data_rules_client.ts

```diff
@@ -408,12 +410,14 @@ export class RuleMigrationsDataRulesClient extends RuleMigrationsDataBaseClient
     {
       status,
       ids,
+      installed,
       installable,
       prebuilt,
-      custom,
       searchTerm,
       failed,
-      notFullyTranslated,
+      fullyTranslated,
+      partiallyTranslated,
+      untranslatable,
```

Comment:
NIT: At this point maybe it's better get all the `filters` in a variable instead of this massive destructing:
```ts
  private getFilterQuery(
    migrationId: string,
    filters: RuleMigrationFilters = {}
  ): QueryDslQueryContainer {
    [...]
```

Reply:
Agree! Will add this in my next PR together with the proposed above changes.